sql/src/test/java/io/crate/analyze/DropFunctionAnalyzerTest.java

```diff
@@ -45,13 +51,36 @@ public void testDropFunctionSimple() throws Exception {
         assertThat(analyzedStatement, instanceOf(DropFunctionAnalyzedStatement.class));
 
         DropFunctionAnalyzedStatement analysis = (DropFunctionAnalyzedStatement) analyzedStatement;
+        assertThat(analysis.schema(), is("doc"));
         assertThat(analysis.name(), is("bar"));
         assertThat(analysis.ifExists(), is(false));
         assertThat(analysis.argumentTypes().get(0), is(DataTypes.LONG));
         assertThat(analysis.argumentTypes().get(1), is(DataTypes.OBJECT));
     }
 
     @Test
+    public void testDropFunctionWithSessionSetSchema() throws Exception {
+        DropFunctionAnalyzedStatement analysis = (DropFunctionAnalyzedStatement) e.analyzer.boundAnalyze(
+            SqlParser.createStatement("DROP FUNCTION bar(long, object)"),
+            new SessionContext(0, Option.NONE, "my_schema"),
+            new ParameterContext(Row.EMPTY, Collections.emptyList())).analyzedStatement();
+
+        assertThat(analysis.schema(), is("my_schema"));
+        assertThat(analysis.name(), is("bar"));
+    }
+
+    @Test
+    public void testDropFunctionWithExplicitSchema() throws Exception {
```

Comment:
use similar name with https://github.com/crate/crate/pull/5225/files#diff-c44088c6f84d9dfbcce76baa0c6ba0acR88 just for consistency :-)